### PR TITLE
Edits to using_openshift/templates.adoc

### PR DESCRIPTION
**Pull request by bfallonf**
Sentence-level edits to said file as per trello card #568. 

@csrwng this comes out from your PR: https://github.com/openshift/openshift-docs/pull/223

Can I ask for a quick tech review? Nothing major has changed, but best to make sure. Thanks!

### CODE REVIEW

**Comment by csrwng** [comment]
@bfallonf LGTM

File: using_openshift/templates.adoc [hunk 1]
```diff
@@ -10,19 +10,19 @@
 toc::[]
 
 == Overview
-A template describes a set of resources intended to be used together that can be customized and processed to produce a configuration. Each template can define a list of parameters that can be modified for consumption by containers. It can also define a set of labels to apply to every resource created by the template.
+A template describes a set of resources intended to be used together that can be customized and processed to produce a configuration. Each template defines a list of parameters that can be modified for consumption by containers. It also defines a set of labels to apply to every resource created by the template.
 
-== Parts of a Template
-.Following is a sample template JSON file:
+.Sample template JSON file:
```
Comment (adellape) [inline using_openshift/templates.adoc]: Title case?

File: using_openshift/templates.adoc [hunk 2]
```diff
@@ -49,20 +49,23 @@ A template describes a set of resources intended to be used together that can be
 <4> A list of labels to apply to resources
 <5> A list of resources to create
 
+====
+
 == Labels
-The list of labels specified in the template will be applied to every resource that is generated from the template. link:../architecture/kubernetes_model.html#label[Labels] can be used to manage and organize generated resources.
+link:../architecture/kubernetes_model.html#label[Labels] are used to manage and organize generated resources, such as pods. The labels specified in the template will be applied to every resource that is generated from the template.
```
Comment (adellape) [inline using_openshift/templates.adoc]: s/will be/are/ ?

File: using_openshift/templates.adoc [hunk 3]
```diff
@@ -49,20 +49,23 @@ A template describes a set of resources intended to be used together that can be
...
 
 == Parameters
-The list of parameters that you want to override are listed in the *parameters* section of the template. Alternatively, you can override them with the following command, specifying the file to be used:
+The list of parameters that you can override are listed in the parameters section of the template. Alternatively, you can override them with the following command, specifying the file to be used:
```
Comment (adellape) [inline using_openshift/templates.adoc]: In "... listed in the parameters section ...", it's referring to the `parameters` section that is part of the template itself, so you could mark it up as:

    `*parameters*`

per [guidelines doc](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) (IMO, fits with "configuration parameter" in this case).

File: using_openshift/templates.adoc [hunk 4]
```diff
@@ -49,20 +49,23 @@ A template describes a set of resources intended to be used together that can be
...
 
 ****
-`osc process --parameters -f [replaceable]#<filename>#`
+`$ osc process --parameters -f _<filename>_`
 ****
 
-The following shows the output when listing the parameters for one of the [replaceable]#openshift/origin/examples# template.
+The following shows the output when listing the parameters for one of the `_openshift/origin/examples_` templates.
```
Comment (adellape) [inline using_openshift/templates.adoc]: I see that `openshift/origin/examples` was marked up as a replaceable in a much earlier commit. So your markup change here would be correct in that case, however I think this was originally meant to refer to the `examples` directory in the `openshift/origin` repo, ie https://github.com/openshift/origin/tree/master/examples. You can see in the example output that follows that the command is processing the `examples/sample-app/application-template-dockerbuild.json` file.

So I think  `openshift/origin/examples` could either be a direct link to the above URL, or just be marked up as a directory instead (bold+italic).

**Comment by adellape** [comment]
@bfallonf Added some inline comments/suggestions. If you do make any additional changes, try squashing your related commits for this feature branch into a single commit with a message generally describing your overall changes (instead of just "initial edits" or the like), before you `push` (with `--force`).

**Comment by bfallonf** [comment]
@csrwng Cool, thanks for that.

@adellape Suggestions taken into account. If that's your peer review, this should be ok to merge.

**Comment by adellape** [comment]
@bfallonf :+1:

**Pull request merged**